### PR TITLE
Feature/security  index

### PR DESCRIPTION
**Pull request by dnaluz**
This PR fixes all errors and all but 2 warnings reported by running 

`phpcs --standard=Wordpress`
`phpcs --standard=PHPCompatibilityWP`

for the following files:

* header.php
* footer.php
* index.php
* front-page.php

The two warnings (same warning) have been reported in the PHPCS tracker spread sheet. 

I had to refactor a couple of things.  The google analytics had to be removed from the header.php file as per the phpcs Wordpress standard.  Also added was a new theme setting field for the google analytics SRI value as testing with our testing analytics ID was throwing an error by the browser.  Contact me regarding getting this setup as I have a hash with the corresponding Google analytics tag.  This will need to set on staging / production as well.

### CODE REVIEW

File: footer.php [hunk 1]
```diff
@@ -1,96 +1,123 @@
...
-                        ); 
-                    ?>
-                    
-                    <?php 
-                        wp_nav_menu(array(                                 
-                                        'menu'           =>  'Mozilla Main Menu',
-                                        'menu_class'        =>  'footer-nav-menu')
-                        ); 
-                    ?>
+
```
Comment (kycapstick) [inline footer.php:15]: This blank line is throwing the following warning: Found precision alignment of 1 spaces.

File: footer.php [hunk 2]
```diff
@@ -1,96 +1,123 @@
...
+				<div class="footer__menu-container">
+					<?php
+						wp_nav_menu(
+							array(
+								'menu'       => 'Footer Primary',
+								'menu_class' => 'footer-menu footer-menu--primary',
+							)
+						);
+						?>
+
```
Comment (kycapstick) [inline footer.php:62]: This blank line is throwing the following warning: Found precision alignment of 1 spaces.

File: front-page.php [hunk 3]
```diff
@@ -198,19 +221,19 @@
...
-										<?php foreach($meta['group_tags'] AS $key =>  $value): ?>
-											<li class="groups__tag"><?php print $value; ?></li>
+										<?php foreach ( $meta['group_tags'] as $key => $value ) : ?>
+											<li class="groups__tag"><?php echo esc_html( $value ); ?></li>
 											<?php $tag_counter++; ?>
-											<?php if($tag_counter === 2 && sizeof($meta['group_tags']) > 2): ?>
-												<li class="groups__tag">+ <?php print sizeof($meta['group_tags']) - 2; ?> <?php print __(' more tags', 'community-portal'); ?></li>
+											<?php if ( 2 === $tag_counter && count( $meta['group_tags'] ) > 2 ) : ?>
+												<li class="groups__tag">+ <?php echo esc_html( count( $meta['group_tags'] ) - 2 ); ?> <?php echo esc_html_e( ' more tags', 'community-portal' ); ?></li>
 												<?php break; ?>
```
Comment (kycapstick) [inline front-page.php:237]: This line throws a warning that code after this break statement can't run

Comment (dnaluz) [reply front-page.php:237]: Yeah this one is fine.

File: footer.php (hunk 2 above)
Comment (dnaluz) [reply footer.php]: Fixed

File: footer.php (hunk 1 above)
Comment (dnaluz) [reply footer.php]: Fixed

**Pull request merged**